src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyData.java

```diff
@@ -0,0 +1,30 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.AccountAttributes;
+
+/**
+ * PageData object that is sent back as a result of InstructorFeedbackEditCopy. 
+ * Specifies a page for the user to be redirected to as a result of the form submission. 
+ *
+ */
+public class InstructorFeedbackEditCopyData extends PageData {
+    public final String redirectUrl;
+    public final String errorMessage;
+    public final boolean isError;
+    
+    
+    public InstructorFeedbackEditCopyData(AccountAttributes account, 
+                                          String redirectUrl, String errorMessage) {
+        super(account);
+        this.redirectUrl = redirectUrl != null ? redirectUrl : "";
+        this.errorMessage = errorMessage;
+        this.isError = !errorMessage.isEmpty();
```

Comment:
Many of our ajax response data do contain an <code>isError</code> field though, and it seems like something we could standardize in ajaxResult in future to use the <code>isError</code> field in Action in future. But sure, I'll remove it.